tests/core/test_memory.py

```diff
@@ -33,6 +33,21 @@ def forward(self, *args, **kwargs):
         return {'loss': self.parameter.sum()}
 
 
+class PreCalculatedModel(LightningModule):
+    """ A module with precalculated total params size in MB. """
+
+    def __init__(self):
+        super().__init__()
+        self.layer1 = nn.Linear(10, 100)
+        self.layer2 = nn.Linear(100, 2)
+        self.pre_calculated_model_size = 0.005
+
+    def forward(self, x):
+        x = self.layer1(x)
+        x = self.layer2(x)
+        return x
```

Comment:
maybe use the existing models already present in the file like `MixedDtypeModel` or `UnorderedModel` since `pre_calculated_model_size` is the only new property defined here.

Reply:
@rohitgr7 using this because other model have their own distinct property like MixedDtypeModel has mixed dtype, etc

hence just for goog readablity added such. :)

Reply:
yeah, not a problem, just a suggestion.

Reply:
@rohitgr7 sure :)